instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/core/SetRoutesTest.kt

```diff
@@ -70,7 +47,7 @@ class SetRoutesTest : BaseTest<EmptyTestActivity>(EmptyTestActivity::class.java)
     }
 
     @Test
-    fun set_navigation_routes_successfully() {
+    fun set_navigation_routes_successfully() = runBlocking {
         val mockRoute = MockRoutesProvider.dc_very_short(activity)
         val route = mockRoute.routeResponse.routes()[0]
```

Comment:
Oh, I didn't notice in previous PR. Let's use `mapboxNavigation.requestRoutes(routeOptions)
                .getSuccessfulResultOrThrowException().routes`

Reply:
Discussed internally, changed the route creation a little bit.